README.md

```diff
@@ -44,45 +70,96 @@ The structure should appear like so:
 \---specification
 |    +---automation
 |    |   \---resource-manager
-|    |       \---Microsoft.Automation
-|    |           \---stable
-|    |               \---2015-10-31
-|    |                   \---examples
+|    |       +---Microsoft.Automation
+|    |       |   \---stable
+|    |       |       \---2015-10-31
+|    |       |           \---examples
+|    |       \---readme.md
 |    +---batch
 |    |   +---data-plane
-|    |   |   \---Microsoft.Batch
-|    |   |       +---stable
-|    |   |       |   +---2015-12-01.2.2
-|    |   |       |   +---2016-02-01.3.0
-|    |   |       |   +---2016-07-01.3.1
-|    |   |       |   +---2017-01-01.4.0
-|    |   |       |       \---examples
-|    |   |       \---preview
-|    |   |           \---2017-05-01.5.0
+|    |   |   +---stable
```

Comment:
I think we want to indicate here that a service _may_ (and generally does) have one or more folders underneath `data-plane` that contain the `preview` and `stable` folders.

Reply:
The support is described in section "Folder Structure for Service Group"